tools/bazel/BUILD.bazel

```diff
@@ -7,13 +7,13 @@ bzl_library(
 )
 
 xcode_version(
-    name = "github_actions_xcode_14_2_0",
-    default_macos_sdk_version = "13.1",
-    version = "14.2",
+    name = "github_actions_xcode_16_2_0",
+    default_macos_sdk_version = "14.5",
+    version = "16.2",
```

Comment:
Note: 16.3 gives you LLVM19, 16.2 is still based on LLVM17. Both are available on the macos-15 image, might be best to change this now for forward compatibility.